python/ucxx/ucxx/_lib/tests/test_cancel.py

```diff
@@ -80,7 +80,6 @@ def test_message_probe():
         args=(queue,),
     )
     client.start()
-    client.join(timeout=10)
-    server.join(timeout=10)
+    join_processes([client + server], timeout=10)
```

Comment:
**Problem**: The `join_processes` function is called with an incorrect argument. The `client + server` expression will result in an error because you cannot add two process objects together.

**Suggested Change**: The `join_processes` function expects a list of processes. To fix this issue, pass the processes as a list.


**Severity (1 - 4)**: 2 - MINOR

**Line**: 83

```suggestion
    join_processes([client, server], timeout=10)
```

-----

<div align="right">
    <small><i>Generated by <b>Llama 3.1 405b</b></i></small>
</div>
<br />


<div align="right">
  <small>Was this helpful?</small>
  <a href="https://codecritic-feedback.nvidia.com?vote=1&comment_guid=f566a190-b259-4779-b307-73eda4221612&is_github=true" target="_blank">
    <button>👍</button>
  </a>
  <a href="https://codecritic-feedback.nvidia.com?vote=-1&comment_guid=f566a190-b259-4779-b307-73eda4221612&is_github=true" target="_blank">
    <button>👎</button>
  </a>
</div>